ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+If you're not sure whether your problem is specifically related to this repository, or to the ConfigCat service overall, please contact the [ConfigCat support](https://configcat.com/support). This issue tracker is intended only for feedback on the content of this repository. 
+
+### Describe the bug
+
+A clear and concise description of what the bug is.
+
+### To reproduce
+
+Steps to reproduce the behavior.
+
+### Expected behavior
+
+A clear and concise description of what you expected to happen.
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+
+## SDK related information
+
+If you are reporting a bug related to one of our SDK's, please provide the following information.
+
+### SDK version
+
+The version of this SDK that you are using.
+
+### SDK configuration
+
+If it differs from the default, add your SDK configuration here e.g. **polling mode**, **refresh interval** or **custom cache**.
+
+### Logs
+
+If applicable, add the related SDK's log output here.
+
+### Language version
```

Comment:
Suggestion: Programming language version

Reply:
I think it's pretty obvious that we want a programming language version here. But I extended with the term `Framework` cause in some cases that's more accurate than the language.